testing/web3/Logs.txt

```diff
@@ -0,0 +1,8400 @@
+"initialblockdownload": false,
```

Comment:
If there's a way to delete this from the commits can we do that plz?

Reply:
Yes, sorry about that. Just removed it and added stuff to the .gitignore